### PR TITLE
fix: redirect urls to .html

### PR DESCRIPTION
**Pull request by justyna-13**
# Related issue

Closes # https://vsf.atlassian.net/jira/software/c/projects/SFUI2/boards/69?modal=detail&selectedIssue=SFUI2-1160&assignee=5fcf86fca1d3f100684db8a5

# Scope of work

# Screenshots of visual changes

https://github.com/vuestorefront/storefront-ui/assets/46591755/82642e92-21fc-4647-a407-287d83d98aba

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: .vuestorefrontcloud/docs/default.conf [hunk 1]
```diff
@@ -7,6 +7,10 @@ server {
   }
 
   location /v2/ {
+    if ($request_uri ~ ^/(.*)\.html(\?|$)) {
+        return 302 /$1;
+    }
```
Comment (FRSgit) [inline .vuestorefrontcloud/docs/default.conf:12]: Is that condition necessary? What does it do? Maybe some comment would be useful 😄

File: .vuestorefrontcloud/docs/default.conf [hunk 2]
```diff
@@ -7,6 +7,10 @@ server {
...
+    try_files $uri $uri.html $uri/ =404;
```
Comment (FRSgit) [inline .vuestorefrontcloud/docs/default.conf:13]: I'm wondering about `$uri/` part - what does it do here?

File: .vuestorefrontcloud/docs/default.conf (hunk 1 above)
Comment (justyna-13) [reply .vuestorefrontcloud/docs/default.conf]: I thought so but I've tried it without it and seems to work

File: .vuestorefrontcloud/docs/default.conf (hunk 2 above)
Comment (justyna-13) [reply .vuestorefrontcloud/docs/default.conf:10]: $uri/ matches the directory but I'm still looking for the latest issue so I'm converting to draft for now

**Review by FRSgit — APPROVED** [review]

**Pull request merged**